index.md

```diff
@@ -0,0 +1,63 @@
+<img width="500" align="right" alt="illustration" src="https://user-images.githubusercontent.com/82679007/150655220-d023b26e-ceab-4b37-9d66-0ad6b20f0628.png">
```

Comment:
Can you use the Markdown syntax here `![Alt text](path/to/file)`? And add the image in the PR?